File: Exam/PHP/src/AppBundle/Entity/Issue.php
<?php

namespace AppBundle\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Issue
 *
 * @ORM\Table(name="issue")
 * @ORM\Entity(repositoryClass="AppBundle\Repository\IssueRepository")
 */
class Issue
{
    /**
     * @var integer
     *
     * @ORM\Column(name="id", type="integer")
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="AUTO")
     */
    public $id;


    /**
     * @var string
     *
     * @ORM\Column(name="title", type="string")
     */
    public $title;

    /**
     * @var string
     *
     * @ORM\Column(name="content", type="string")
     */
    public $content;

    /**
     * @var integer
     *
     * @ORM\Column(name="priority", type="string")
     */
    public $priority;

    /**
     * @return int
     */
    public function getId()
    {
        return $this->id;
    }

    /**
     * @param int $id
     */
    public function setId($id)
    {
        $this->id = $id;
    }

    /**
     * @return string
     */
    public function getTitle()
    {
        return $this->title;
    }

    /**
     * @param string $title
     */
    public function setTitle($title)
    {
        $this->title = $title;
    }

    /**
     * @return string
     */
    public function getContent()
    {
        return $this->content;
    }

    /**
     * @param string $content
     */
    public function setContent($content)
    {
        $this->content = $content;
    }

    /**
     * @return int
     */
    public function getPriority()
    {
        return $this->priority;
    }

    /**
     * @param int $priority
     */
    public function setPriority($priority)
    {
        $this->priority = $priority;
    }



}

File: Exam/Java/src/main/java/issuetracker/entity/Issue.java
package issuetracker.entity;

import javax.persistence.*;

@Entity
@Table(name = "issues")
public class Issue {

    public Integer Id;
    public String Title;
    public String Content;
    public Integer Priority;

    public Issue(String title, String content, Integer priority){
        this.Title = title;
        this.Content = content;
        this.Priority = priority;
    }

    public Issue(){ }


    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    public Integer getId() {
        return Id;
    }

    public void setId(Integer id) {
        Id = id;
    }

    @Column(nullable = false)
    public String getTitle() {
        return Title;
    }

    public void setTitle(String title) {
        Title = title;
    }


    @Column(nullable = false)
    public String getContent() {
        return Content;
    }

    public void setContent(String content) {
        Content = content;
    }


    @Column(nullable = false)
    public Integer getPriority() {
        return Priority;
    }

    public void setPriority(Integer priority) {
        Priority = priority;
    }
}
